index.html

```diff
@@ -63,6 +63,171 @@ <h1 class="heading1">I’m Gedeon<br />Glad to see<br />you!</h1>
+            sign-ups required.
+          </p>
+          <ul class="ulc2">
+            <li><a href="#" class="lis">HTML</a></li>
+            <li><a href="#" class="lis">CSS</a></li>
+            <li><a href="#" class="lis">JAVASCRIPT</a></li>
+          </ul>
+          <a href="#" class="btn1">See Project</a>
+        </section>
+        <section class="cards card3">
+          <img
+            src="IMG/Snapshoot-Portfolio3.svg"
+            alt="portfolio snapshot"
+            class="snapshot1"
+          />
+          <h2 class="tonic">Tonic</h2>
+          <ul class="ulc1">
+            <li class="li1"><a href="#">CANOPY</a></li>
+            <li class="counter"><img src="IMG/Counter.svg" alt="." /></li>
+            <li class="li2"><a href="#">Back End Dev</a></li>
+            <li class="counter"><img src="IMG/Counter.svg" alt="." /></li>
+            <li class="li3"><a href="#">2015</a></li>
+          </ul>
+          <p class="par3">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="ulc2">
+            <li><a href="#" class="lis">HTML</a></li>
+            <li><a href="#" class="lis">CSS</a></li>
+            <li><a href="#" class="lis">JAVASCRIPT</a></li>
+          </ul>
+          <a href="#" class="btn1">See Project</a>
+        </section>
+        <section class="cards card4">
+          <img
+            src="IMG/Snapshoot-Portfolio4.svg"
+            alt="portfolio snapshot"
+            class="snapshot1"
+          />
+          <h2 class="tonic">
+            Multi-post<br />
+            stories
+          </h2>
+          <ul class="ulc1 upost">
+            <li class="li1"><a href="#">CANOPY</a></li>
+            <li class="counter"><img src="IMG/Counter.svg" alt="." /></li>
+            <li class="li2"><a href="#">Back End Dev</a></li>
+            <li class="counter"><img src="IMG/Counter.svg" alt="." /></li>
+            <li class="li3"><a href="#">2015</a></li>
+          </ul>
+          <p class="par3">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="ulc2">
+            <li><a href="#" class="lis">HTML</a></li>
+            <li><a href="#" class="lis">CSS</a></li>
+            <li><a href="#" class="lis">JAVASCRIPT</a></li>
+          </ul>
+          <a href="#" class="btn1">See Project</a>
+        </section>
+      </section>
+      <section class="abt">
+        <section class="myself abt1">
+          <h2 class="heading1 hd2">About<br />Myself</h2>
+          <p class="par1">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,don’t hestiate to
+            contact me.
+          </p>
```

Comment:
- [ ] [OPTIONAL] Consider removing the underlining effect on the text that is not a link.
<img width="383" alt="Screen Shot 2022-03-18 at 13 43 11" src="https://user-images.githubusercontent.com/28307960/158998420-23029ed3-fa0a-4f32-a15d-782d5d53926a.png">